docs/product-installation-guide/src/main/asciidoc/jms-factories-ref.adoc

```diff
@@ -23,10 +23,6 @@ endif::BA[]
 | Receiving all responses produced by {KIE_SERVER}
 
 ifdef::BA[]
-|`KIE.RESPONSE.ALL`
-|`jms/cf/KIE.RESPONSE.ALL`
-| Receiving all responses produced by {PRODUCT}
-
 |`KIE.SESSION`
```

Comment:
@mswiderski, can you validate? Nothing against @sutaakar :). I'm just surprised it didn't come up before. This would mean that all JMS sections in the doc and in other docs dealing with connection factories, queues, and activation specifications (as Karel indicates) would not have 7 or 8 requirements but would have apparently only one: KIE.SERVER.EXECUTOR. This will also impact a lot of other areas of the document, where system properties are listed (such as JAVA_OPTIONS variable, etc.) so I just want to be totally sure.

Reply:
looking at all queues defined here:
KIE.SERVER.REQUEST
KIE.SERVER.RESPONSE
KIE.SESSION
KIE.TASK
KIE.AUDIT
KIE.SIGNAL
KIE.SERVER.EXECUTOR

on KIE.SESSION and KIE>TASK should be removed other should be still there though KIE.AUDIT KIE.SIGNAL are certainly optional (might be good to rename them to KIE.SERVER.SIGNAL and KIE.SERVER.AUDIT)
here are the queue KIE Server expects by default https://github.com/kiegroup/droolsjbpm-integration/blob/master/kie-server-parent/kie-server-wars/kie-server/src/main/ee7-resources/META-INF/kie-server-jms.xml

or did I misunderstood it?

Reply:
I see. @mswiderski and @sutaakar, since this is focused on the *required* JMS queues/factories/specs/etc., then it makes sense to list only the following, per Maciej's clarification:

KIE.SERVER.REQUEST
KIE.SERVER.RESPONSE
KIE.SERVER.EXECUTOR

And the same for the other impacted areas (queues/factories/specs/etc.)

That work for you two? This is probably all Karel was saying. My apologies if I misunderstood.

Reply:
fine with me

Reply:
Done. Thanks!